rakelib/compliance.rake

```diff
@@ -0,0 +1,30 @@
+require 'json'
+require 'json-schema'
+
+task :validate => 'compliance:schema:validate'
+
+namespace :compliance do
+  namespace :schema do
```

Comment:
`compliance:schema:` still seems a little generic for the overall rake task list.

What about something more specific, like `compliance_markup:schema` or `compliance:profile:schema`?